autoload/ale/completion.vim

```diff
@@ -678,10 +722,12 @@ function! ale#completion#Done() abort
     let s:last_done_pos = getpos('.')[1:2]
 endfunction
 
+augroup ALECompletionActions
+    autocmd CompleteDone * call ale#completion#HandleUserData(v:completed_item)
+augroup END
+
 function! s:Setup(enabled) abort
     augroup ALECompletionGroup
-        autocmd!
```

Comment:
Put this back again. Every completion group should start by clearing out previous `autocmd` lines.

Reply:
Roger, I removed this by accident.